src/pytorch_lightning/plugins/io/checkpoint_plugin.py

```diff
@@ -43,7 +43,7 @@ def save_checkpoint(self, checkpoint: Dict[str, Any], path: _PATH, storage_optio
         """
 
     @abstractmethod
-    def load_checkpoint(self, path: _PATH, map_location: Optional[Any] = None) -> Dict[str, Any]:
+    def load_checkpoint(self, path: _PATH, map_location: MAP_LOCATION = None) -> Dict[str, Any]:
```

Comment:
same here

Reply:
reverted to https://github.com/Lightning-AI/lightning/pull/13932/commits/0502a4e9fe2f1b229265de8ae82a129e704f65ee and synced with upstream in https://github.com/Lightning-AI/lightning/pull/13932/commits/82fc4528431fd3412e30782ba64bcc65d2adba38